psamm/commands/gapfill.py

```diff
@@ -99,3 +110,9 @@ def run(self):
                 print('{}\t{}\t{}'.format(rxnid, 'Reverse', rxt))
         else:
             logger.info('No blocked compounds found')
+
+    def _log_epsilon_and_fail(self, epsilon, exc):
+        msg = ('Finding blocked compounds failed with epsilon set to {}. Try'
+               ' lowering the epsilon value to reduce artifical constraints on'
+               ' the model.'.format(epsilon))
+        self.fail(msg, exc)
```

Comment:
If this is related to the parameter provided by user, then why this is not a argument error, but a failure, though it is a runtime failure. Or the point is, this is a failure, but user provide what needs to be provided, it's just the epsilon to high or something, so we don't want to show them the argument help information again?